framework/test/unit/modules/pos/commands/report_misbehavior.spec.ts

```diff
@@ -528,6 +528,27 @@ describe('ReportMisbehaviorCommand', () => {
 
 			await expect(pomCommand.verify(context)).not.toReject();
 		});
+
+		it('should return error when generator of header1 is not a validator', async () => {
+			transactionParamsDecoded = {
+				header1: codec.encode(blockHeaderSchema, {
+					...header1,
+					generatorAddress: utils.getRandomBytes(32),
+				}),
+				header2: codec.encode(blockHeaderSchema, { ...header2 }),
+			};
+			transaction.params = codec.encode(pomCommand.schema, transactionParamsDecoded);
+			context = testing
+				.createTransactionContext({
+					stateStore,
+					transaction,
+				})
+				.createCommandExecuteContext<PomTransactionParams>(pomCommand.schema);
+
+			await expect(pomCommand.verify(context)).rejects.toThrow(
+				'Different generator address never contradict to each other.',
```

Comment:
here it's failing because header1 and header2 has different generatorAddress, try to use the same address for header2 as well